domains.txt

```diff
@@ -411,11 +459,14 @@ masoncole.co.uk
 masonfrank.com
 matchtech.com
 maysonhunt.com
+mbauk.com
 mbnsolutions.com
 mccabebarton.com
 mcg-recruitment.com
 mcgregor-boyall.com
+mcpartners.co.uk
```

Comment:
M.C. Partners is an executive search firm that connects forward-thinking businesses and individuals. We are based in London and New York.